### ISSUE TITLE
has no exported member 'RedisOptions'

### ISSUE THREAD

**Issue by devsaran** [upstash/upstash-redis#342]
Hi there,

I am getting the following error while building. 

```
node_modules/@upstash/redis/types/platforms/nodejs.d.ts:33:10 - error TS2694: Namespace '"/functions/node_modules/@upstash/redis/types/pkg/redis"' has no exported member 'RedisOptions'.

33 } & core.RedisOptions & RequesterConfig;
            ~~~~~~~~~~~~


Found 1 error in node_modules/@upstash/redis/types/platforms/nodejs.d.ts:33
```

Node Version: v16.18.1

**Comment by chronark** [comment]
What version are you using?

**Comment by devsaran** [comment]
till version 1.19.3, no problem. When I increment the version. I get this error while building.

**Comment by chronark** [comment]
thanks,
can you share your typescript config as well please?
I can't reproduce this right now.

**Comment by devsaran** [comment]
Here is the Typescript config.

```
{
  "compilerOptions": {
    "module": "commonjs",
    "typeRoots": ["node_modules/@types", "./typings"],
    "noImplicitReturns": true,
    "noImplicitAny": false,
    "noUnusedLocals": true,
    "outDir": "lib",
    "sourceMap": true,
    "strict": true,
    "target": "es2017"
  },
  "compileOnSave": true,
  "include": [
    "src"
  ]
}

```

Here is the code
```
import { Redis } from '@upstash/redis/with-fetch';
import * as functions from 'firebase-functions';

const redis = new Redis({
  url: functions.config()?.env?.config?.REDIS_REST_URL,
  token: functions.config()?.env?.config?.REDIS_REST_TOKEN,
  retry: {
    retries: 1,
  },
});

export default redis;
```

We use it in firebase-functions project.

**Comment by chronark** [comment]
great, I'll try this in the afternoon, thank you

**Comment by devsaran** [comment]
any update?

**Comment by chronark** [comment]
Fixed it in [v1.20.4](https://github.com/upstash/upstash-redis/releases/tag/v1.20.4)
